appender_test.go

```diff
@@ -28,7 +27,8 @@ const (
 		float REAL,
 		double DOUBLE,
 		string VARCHAR,
-		bool BOOLEAN
+		bool BOOLEAN,
+-- 	    blob BLOB
```

Comment:
There previously was no test for `BLOB` types, currently it is causing a segmentation fault in 
`appender.go::443`:
```go
state = C.duckdb_append_data_chunk(*a.appender, chunk)
```

Reply:
@marcboeker I've looked into this and I don't think `BLOB` types have ever fully been supported. Running this test on the current main causes a segmentation fault. To my understanding `BLOB`'s at their core look like this: `[]uint8` in golang, they overlap with slices of `uint8`, which, if I'm not missing something, means we have to choose which one to support.

I've opened an issue -> #152